js/app/utils.js

```diff
@@ -25,6 +25,7 @@ const Gettext = imports.gettext;
 const Gio = imports.gi.Gio;
 const GLib = imports.gi.GLib;
 const Gtk = imports.gi.Gtk;
+const SoupURI = imports.gi.Soup.URI;
```

Comment:
This is a bit unusual. I don't prefer it because then if you later need something else from Soup you have to go back and change the declaration, but if you have a good reason for it I'm open to convincing.